packages/collapsible/src/LionCollapsible.js

```diff
@@ -0,0 +1,127 @@
+import { LitElement, html, css } from '@lion/core';
+
+const uuid = () => Math.random().toString(36).substr(2, 10);
+/**
+ * # <lion-collapsible> webcomponent
+ *
+ * @customElement lion-collapsible
+ * @extends LitElement
+ */
+export class LionCollapsible extends LitElement {
+  static get properties() {
+    return {
+      expanded: {
+        type: Boolean,
+        reflect: true,
+      },
+      transitioning: {
+        type: Boolean,
+        reflect: true,
+      },
+    };
+  }
+
+  constructor() {
+    super();
+    this.expanded = false;
+    this.transitioning = false;
+  }
+
+  static get styles() {
+    return [
+      css`
+        :host {
+          --animation: opacity 0.3s ease-out, max-height 0.3s ease-out;
+          display: block;
+        }
+
+        :host ::slotted([slot='content']) {
+          transition: var(--animation);
+          max-height: 0;
+          opacity: 0;
+          overflow: hidden;
+        }
+
+        :host([expanded]) ::slotted([slot='content']) {
+          max-height: 100vh;
```

Comment:
this would make the animation speed depend on screen size. You could take a look at https://github.com/PolymerElements/iron-collapse/blob/master/iron-collapse.js which dynamically calculates the height of the content. And it uses display: none so you don't need extra tricks for a11y like @erikkroes mentioned :)

Reply:
Thanks for sharing it. I made the changes similarly. :)